samples/valueentity-counter/pom.xml

```diff
@@ -0,0 +1,364 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.example</groupId>
```

Comment:
Should we align with the other samples and use `com.akkaseverless.samples`?

Reply:
No, we shouldn't namespace examples under `com.akkaserverless`. That would easily hide the user's log messages.

Reply:
good point, I will adapt the other samples.